controller/templates/confd_settings.py

```diff
@@ -10,6 +10,9 @@
 except:
     SCHEDULER_OPTIONS = {}
 
+# scheduler swarm manager host
+SWARM_HOST = '{{ if exists "/deis/scheduler/swarm/host" }}{{ getv "/deis/scheduler/swarm/host" }}{{ else }}127.0.0.1{{ end }}'
+
 # base64-encoded SSH private key to facilitate current version of "deis run"
```

Comment:
@gabrtv @carmstrong I want to keep the separate key spaces in the same PR . Yesterday I forgot to mention the two use cases where we require separate key spaces for schedulers.
##### User Forgot to set the config for schedulerModule
1. user ran ```deisctl install <scheduler> && deisctl start <scheduler>``` .
2. The scheduler will keep publishing the ```/deis/controller/schedulerTarget``` to scheduler endpoint. But still uses fleet with the endpoint of the new scheduler 

##### User Forgot to stop the scheduler
1. if a user forgots to stop scheduler and set the config for schedulerModule to a different scheduler 
2. Both the schedulers try to set ```deis/controller/schedulerTarget``` and confd keeps updating controller confd templates accordingly.

 I myself observed these errors, Yesterday I could recollect them